src/OpenTelemetry.Exporter.NewRelic/NewRelicTraceExporter.cs

```diff
@@ -120,6 +120,19 @@ public override ExportResult Export(in Batch<Activity> batch)
             }
         }
 
+        private static string? ActivityKindToString(ActivityKind kind)
+        {
+            return kind switch
+            {
+                ActivityKind.Consumer => "CONSUMER",
+                ActivityKind.Client => "CLIENT",
+                ActivityKind.Internal => "INTERNAL",
+                ActivityKind.Producer => "PRODUCER",
+                ActivityKind.Server => "SERVER",
```

Comment:
How would you feel about adding the strings to the consts that we have?  It'd be cool for us someday to be able to have them in a single place for other Telemetry SDK Users